backend/src/main/resources/application.properties

```diff
@@ -109,16 +109,16 @@ quarkus.unleash.url=http://localhost:4242
 quarkus.log.category."com.redhat.cloud.notifications.routers.internal.kessel.KesselAssetsMigrationService".min-level=TRACE
 quarkus.log.category."com.redhat.cloud.notifications.auth.kessel".min-level=TRACE
 
-inventory-api.authn.client.id=insights-notifications
+inventory-api.authn.client.id=svc-test
 inventory-api.authn.client.issuer=http://localhost:8084/realms/redhat-external
-inventory-api.authn.client.secret=development-value-123
+inventory-api.authn.client.secret=h91qw8bPiDj9R6VSORsI5TYbceGU5PMH
 inventory-api.authn.mode=oidc-client-credentials
 inventory-api.is-secure-clients=false
 inventory-api.target-url=${clowder.endpoints.kessel-inventory-api:localhost:9081}
 
-relations-api.authn.client.id=insights-notifications
+relations-api.authn.client.id=svc-test
 relations-api.authn.client.issuer=http://localhost:8084/realms/redhat-external
-relations-api.authn.client.secret=development-value-123
+relations-api.authn.client.secret=h91qw8bPiDj9R6VSORsI5TYbceGU5PMH
```

Comment:
These are the default values when launching the Kessel's Inventory API with the authentication enabled: https://github.com/project-kessel/inventory-api/blob/e0fc00b00af6af4977f8b4c66230a34f850da4a2/scripts/get-token.sh

In order to simplify things, I think it's just better to leave the credentials like this by default. It eases development and debugging, and since the credentials are public facing, I think it is just safe to have them here.